content/api/trafiklab-apis/gtfs-sweden/realtime.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Realtime data
+aliases:
+  - "api/gtfs-regional-realtime"
```

Comment:
These were only used as a fallback from the old Trafiklab website, and are not needed on new APIs